katalog/grafana/dashboards/nodes-darwin.json

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
Is this dashboard configmap created somewhere?

Reply:
> Is this dashboard configmap created somewhere?

I'm sure it was wrong before because I have verified it and now it is a valid JSON

Reply:
I think @nutellinoit meant that this json file is not being used.

Grafana loads the dashboards from configmaps with a specific label. This json file is not being put inside a configmap, so you won't see the dashboard in Grafana.

See what we do for the other dashboard json file in the kustomization.yaml:
https://github.com/sighupio/fury-kubernetes-monitoring/blob/46f50fd47fd5c7df7c533b11e4a50043707e2d86/katalog/grafana/dashboards/kustomization.yaml#L16-L19

Anyway, from the name seems this dashboard seems to be for Darwin nodes (macos), we don't have those. I'd drop this file if that's the case.